app/components/trade-leads-search-result.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+  actions: {
+    toggleBody: function() {
+      this.toggleProperty('isShowingBody');
+    }
+  }
+});
```

Comment:
Did you ever notice that your editor doesn't add newline characters to the last line of source files? It's better if it does. [Read about it here](http://stackoverflow.com/a/16224292) if you're interested.